Lua Error when using the ACF Menu tool without anything selected

Doesn't seem like a pressing issue by any means. 

When you left click with the ACF Menu tool selected, it throws an error. (Right clicking does not)

```
22:27:08 Lua Error: [ERROR] addons/acf/lua/weapons/gmod_tool/stools/acfmenu.lua:66: attempt to index a nil value 1. LeftClick - addons/acf/lua/weapons/gmod_tool/stools/acfmenu.lua:66 2. unknown - gamemodes/sandbox/entities/weapons/gmod_tool/shared.lua:254
```

When nothing is selected, `ACF.Weapons[Type][Id]` is nil, so trying to index ["ent"] on it throws errors.

--- a/lua/weapons/gmod_tool/stools/acfmenu.lua
+++ b/lua/weapons/gmod_tool/stools/acfmenu.lua
@@ -62,9 +62,9 @@ function TOOL:LeftClick( trace )
 	local ply = self:GetOwner()
 	local Type = self:GetClientInfo( "type" )
 	local Id = self:GetClientInfo( "id" )
-
-        local TypeId = ACF.Weapons[Type][Id]
-        if not TypeId then return false end
+	
+	local TypeId = ACF.Weapons[Type][Id]
+	if not TypeId then return false end
 	
 	local DupeClass = duplicator.FindEntityClass( TypeId["ent"] ) 
 	

--- a/lua/weapons/gmod_tool/stools/acfmenu.lua
+++ b/lua/weapons/gmod_tool/stools/acfmenu.lua
@@ -62,8 +62,11 @@ function TOOL:LeftClick( trace )
 	local ply = self:GetOwner()
 	local Type = self:GetClientInfo( "type" )
 	local Id = self:GetClientInfo( "id" )
+
+        local TypeId = ACF.Weapons[Type][Id]
+        if not TypeId then return false end
 	
-	local DupeClass = duplicator.FindEntityClass( ACF.Weapons[Type][Id]["ent"] ) 
+	local DupeClass = duplicator.FindEntityClass( TypeId["ent"] ) 
 	
 	if DupeClass then
 		local ArgTable = {}

